rna/rna-alignment/rna-alignment.inputs.json

```diff
@@ -13,7 +13,8 @@
   "RNAAlignment.runtime_zones": "us-central1-a us-central1-b",
   "RNAAlignment.star_vm_memory": "80.0GB",
   "RNAAlignment.star_vm_disk_size_gb": 200,
-  
-  "##_COMMENT4": "STAR may take more than 24 hours, so use non-preemptible",
-  "RNAAlignment.preemptible_tries": 0
+
+  "##_COMMENT4": "STAR does not *typically* take more than 24 hours, so use preemptible",
+  "RNAAlignment.preemptible_tries": 3,
+  "RNAAlignment.star_timeout_hours": 24
```

Comment:
Please make the timeout 23.5.
It needs to account for localization/delocalization time.
We found that a half hour buffer was pretty spot on.